portfolio/src/main/webapp/script.js

```diff
@@ -97,10 +98,12 @@ function loadContacts(){
     contacts.forEach((contact) => {
       taskListElement.appendChild(createContactElement(contact));
      
-    })
+    });
   });
 }
 function createContactElement(contact) {
+  contactIds.push(contact.id);
+
   const contactElement = document.createElement('li');
   contactElement.className = 'contact';
 
```

Comment:
Please make these literals constants at the top of this file. e.g.
let EMAIL_ELEMENT_PREFIX = "Email"
then here you can do:
email.Element.innerText = EMAIL_ELEMENT_PREFIX + ": " + contact.email;  or you can likewise try "".concat(EMAIL_PREFIX_TEXT, ": ", contact.email) for a more functional solution, and to avoid creating new objects.